source/includes/_08-additional-features.md.erb

```diff
@@ -479,4 +480,115 @@ should be surfaced using the <code>field_checks</code> attribute of the <code>ch
+
+The identifier for this feature is either <code>EXTERNAL_LINK</code> or <code>EXTERNAL_EMBED</code> depending on how you intend to surface the external resource. This feature is compatible with all of our supported check types.
+
+The `external_resources` object includes whether the external resource should be displayed as a link or an embed, the url, id, and a label describing the resource. 
+
+
+>This endpoint should return JSON in the following format:
+
+```json
+<%=pretty_json("responses/run_check_sync_company_custom", "json") %>
+```
+
+<table>
+  <thead>
+    <th>Field</th>
+    <th>Type</th>
+    <th>Required?</th>
+    <th>Description</th>
+  </thead>
+  <tbody>
+    <tr>
+      <td><code>type</code></td>
+      <td>"EMBED"</a></td>
+      <td>No</td>
+      <td>The type of external resource that is being reported.</td>
+    </tr>
+    <tr>
+      <td><code>url</code></td>
+      <td>string</code></td>
+      <td>No</td>
+      <td>Provides additional detail for certain types of error.</td>
+    </tr>
+    <tr>
+      <td><code>id</code></td>
+      <td>string</td>
+      <td>Yes</td>
+      <td>
+        The external_resource identifier
+      </td>
+    </tr>
+    <tr>
+      <td><code>label</code></td>
+      <td>string</td>
+      <td>No</td>
+      <td>
+        Integration-specific structured error information that customers can
+        use to get additional information about the error being reported.
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+### Authentication
+
+Passfort will make a request to your application via a signed url, which will contain arguments as query parameters:
+
+<code>https://your-url.com/<your-id>?version=<version>&valid_until=<valid_until>&auditee_id=<auditee_id>&signature=<signature></code>
+
+The list of query parameter arguments is shown below, and is delivered in this order:
+
+<table>
+  <thead>
+    <th>Field</th>
+    <th>Type</th>
+    <th>Required?</th>
+    <th>Description</th>
+  </thead>
+  <tbody>
+    <tr>
+      <td><code>custom_data</code></td>
+      <td>string</td>
+      <td>No</td>
+      <td>Any other parameters the integration wishes to have provided</td>
+    </tr>
+    <tr>
+      <td><code>version</code></td>
+      <td>string</td>
+      <td>Yes</td>
+      <td>Versioning information for the signed URL.</td>
+    </tr>
+    <tr>
+      <td><code>valid_until</code></td>
+      <td>string</td>
+      <td>Yes</td>
+      <td>The epoch time the URL is considered valid until (requests after this time should be deemed unauthentic)</td>
+    </tr>
+    <tr>
+      <td><code>auditee_id</code></td>
+      <td>string</td>
+      <td>Yes</td>
+      <td>UUID identifying the user or API key viewing the check</td>
+    </tr>
+    <tr>
+      <td><code>signature</code></td>
+      <td>string</td>
+      <td>Yes</td>
+      <td>A base64url encoding of the SHA-256 HMAC of the entire URL, excluding the signature param</td>
```

Comment:
Is "signature param" a common term or should we change it to "signature parameter"?

Reply:
Period